providers/zohocrm/errors.go

```diff
@@ -0,0 +1,45 @@
+package zohocrm
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"net/http"
+)
+
+/*
+The normal standard response from ZohoCRM look like this:
+{
+	"data": [{...}, {...}],
+	"info": {...}
+}
+*/
+
+// responseHandler wraps the http.StatusNotModified response into http.StatusOK
+// and sends back an empty data field response.
+func responseHandler(resp *http.Response) (*http.Response, error) { //nolint:cyclop
+	// responseData represents data we will send back when the response status code 304.
+	responseData := map[string]any{
+		"data": []any{},
```

Comment:
so this is not data. This is the object name in lowercase.

Reply:
This should be resolved with the above comment, If Accepted.